File: bioweb/src/java/entity/Producto.java
package entity;

/**
 *
 * @author Crispancho
 */
public class Producto {
    
    private int idProducto;
    private String codigo;
    private String nombre;
    private String observacion;

    public Producto() {
    }

    public Producto(int idProducto, String codigo, String nombre, String observacion) {
        this.idProducto = idProducto;
        this.codigo = codigo;
        this.nombre = nombre;
        this.observacion = observacion;
    }

    
    
    public int getIdProducto() {
        return idProducto;
    }

    public void setIdProducto(int idProducto) {
        this.idProducto = idProducto;
    }

    public String getCodigo() {
        return codigo;
    }

    public void setCodigo(String codigo) {
        this.codigo = codigo;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getObservacion() {
        return observacion;
    }

    public void setObservacion(String observacion) {
        this.observacion = observacion;
    } 

    @Override
    public String toString() {
        return "Producto{" + "idProducto=" + idProducto + ", codigo=" + codigo + ", nombre=" + nombre + ", observacion=" + observacion + '}';
    }
    
    
}

File: bioweb/entity/Producto.php
<?php

/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 * Description of Producto
 *
 * @author crispancho
 */
class Producto {
    private $idProducto;
    private $codigo;
    private $nombre;
    private $observacion;
    
    function __construct() {
        
    }
    
    function getIdProducto() {
        return $this->idProducto;
    }

    function getCodigo() {
        return $this->codigo;
    }

    function getNombre() {
        return $this->nombre;
    }

    function getObservacion() {
        return $this->observacion;
    }

    function setIdProducto($idProducto) {
        $this->idProducto = $idProducto;
    }

    function setCodigo($codigo) {
        $this->codigo = $codigo;
    }

    function setNombre($nombre) {
        $this->nombre = $nombre;
    }

    function setObservacion($observacion) {
        $this->observacion = $observacion;
    }


}
